app/components/Code/Code.less

```diff
@@ -3,14 +3,23 @@
 .root {
   .rawText(@standard-type-scale);
   position: relative;
-  padding: (@grid-row-height * 2) (@grid-gutter-width * 2);
+  padding: (@grid-row-height * 2) 0;
```

Comment:
Ah I see you moved the padding to `.card`. Still, I think the 2 gutter spacing is too much, cuts off more code than it needs to. See [next comment](https://github.com/SEEK-Jobs/seek-style-guide/pull/57/files#r84386417)